app/views/submissions/steps/trial/_step02.html.haml

```diff
@@ -1,17 +1,17 @@
 .alert.alert-info
   Before uploading your trait score sheet, please choose
-  %b plant traits
+  %b existing trait descriptors
   that were scored in the course of this trial,
   or define
   %b new trait descriptors.
   Further instructions are found below.
-  Examples of plant traits in the database are: 'oleic acid content' or ' seed mature time'
+  Examples of plant traits in the database are: 'oleic acid content' or ' seed mature time'.
 
 .form-group
   = content.label :trait_descriptor_list
-  - options = options_for_submission_select(@content.existing_trait_descriptors, :descriptor_name)
-  - options.concat options_for_submission_select(@content.new_trait_descriptors, :descriptor_name, id: false)
-  = content.select :trait_descriptor_list, options.html_safe, {},
+  -#- options = options_for_submission_select(@content.existing_trait_descriptors, :descriptor_name)
+  -#- options.concat options_for_submission_select(@content.new_trait_descriptors, :descriptor_name, id: false)
```

Comment:
No, I have not investigated any further.

2016-05-29 12:54 GMT+02:00 Tomasz Gubała <notifications@github.com>:

> In app/views/submissions/steps/trial/_step02.html.haml
> <https://github.com/eSpectrum-IT/brassica/pull/516#discussion_r65001831>:
>
> >
> >  .form-group
> >    = content.label :trait_descriptor_list
> > -  - options = options_for_submission_select(@content.existing_trait_descriptors, :descriptor_name)
> > -  - options.concat options_for_submission_select(@content.new_trait_descriptors, :descriptor_name, id: false)
> > -  = content.select :trait_descriptor_list, options.html_safe, {},
> > +  -#- options = options_for_submission_select(@content.existing_trait_descriptors, :descriptor_name)
> > +  -#- options.concat options_for_submission_select(@content.new_trait_descriptors, :descriptor_name, id: false)
>
> I'm sorry for stating the obvious. I'll try to have a look at it, unless
> you do that already?
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/eSpectrum-IT/brassica/pull/516/files/0f131c697bf2c02bde1300456dfc12db84cbd4d6#r65001831>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAA2ye4txVRR9OBDtZi_zsTTWpB8f7Raks5qGXBogaJpZM4IpK03>
> .
>



-- 
Tomasz Szymczyszyn